sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/CxxPlugin.java

```diff
@@ -340,16 +341,6 @@
       .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
       .index(3)
       .build(),
-      PropertyDefinition.builder(CxxCoverageSensor.FORCE_ZERO_COVERAGE_KEY)
```

Comment:
@jmecosta not sure but I thing I read that FORCE_ZERO_COVERAGE is a core feature now? To be checked...